### PR TITLE
Upgrade to 369, remove deprecated settings and enable setting arbitrary config

### LINKED ISSUE #102: Support for extra config
I need to add auth config properties that's not available in the chart, like these: [Trino OAuth 2.0 authentication](https://trino.io/docs/current/security/oauth2.html)

I've been reading the templates but haven't found a way to inject arbitrary configuration or issues mentioning

### PR DESCRIPTION
**Pull request by esselius**
Any style inputs are appreciated! Just needed to get this working :)

I had upgrade to 368, the most recent release that worked out of the box with the helm chart, but was hit by this bug: https://github.com/trinodb/trino/issues/10710, which ofc was introduced in 367 :)

Fixes #102

### CODE REVIEW

**Comment by valeriano-manassero** [comment]
Can you pls run `helm-docs` and regenerate README? This should be enough to pass tests.

**Comment by esselius** [comment]
There you go :)

**Comment by valeriano-manassero** [comment]
Thank you @esselius ; there are stil la couple of comments in your code that needs to be addressed; can you pls check? Ty.

**Comment by esselius** [comment]
There you go :)

**Comment by valeriano-manassero** [comment]
@esselius pls check Pending items in my review; especially about not changing Presto folder files and old appVersion in Trino chart. When every item is solved we will be ready to launch CI; atm they will not pass.

Can you pls fix them?

**Comment by esselius** [comment]
> @esselius pls check Pending items in my review; especially about not changing Presto folder files and old appVersion in Trino chart. When every item is solved we will be ready to launch CI; atm they will not pass.
> 
> Can you pls fix them?

Sorry been busy 😅 I'm not sure I follow, but I allowed you access so you can fix whatever's needed, if I don't get there first :)

File: valeriano-manassero/presto/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "348"
 description: Deprecated, please use new Trino chart
 name: presto
-version: 1.2.9
+version: 1.2.10
```
Comment (esselius) [inline valeriano-manassero/presto/Chart.yaml:5]:
```suggestion
version: 1.2.9
```

File: valeriano-manassero/presto/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # presto
 
-![Version: 1.2.9](https://img.shields.io/badge/Version-1.2.9-informational?style=flat-square) ![AppVersion: 348](https://img.shields.io/badge/AppVersion-348-informational?style=flat-square)
+![Version: 1.2.10](https://img.shields.io/badge/Version-1.2.10-informational?style=flat-square) ![AppVersion: 348](https://img.shields.io/badge/AppVersion-348-informational?style=flat-square)
```
Comment (esselius) [inline valeriano-manassero/presto/README.md:3]:
```suggestion
![Version: 1.2.9](https://img.shields.io/badge/Version-1.2.9-informational?style=flat-square) ![AppVersion: 348](https://img.shields.io/badge/AppVersion-348-informational?style=flat-square)
```

File: valeriano-manassero/trino/Chart.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "365"
```
Comment (esselius) [inline valeriano-manassero/trino/Chart.yaml:2]:
```suggestion
appVersion: "369"
```

**Comment by esselius** [comment]
> > @esselius pls check Pending items in my review; especially about not changing Presto folder files and old appVersion in Trino chart. When every item is solved we will be ready to launch CI; atm they will not pass.
> > Can you pls fix them?
> 
> Sorry been busy 😅 I'm not sure I follow, but I allowed you access so you can fix whatever's needed, if I don't get there first :)

Ah I think I got it now 🦀

**Review by valeriano-manassero — APPROVED** [review]

**Comment by valeriano-manassero** [comment]
Ty @esselius , PR merged and release is ongoing!

**Pull request merged**